<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>683</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>896</width>
    <height>683</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Belden Network Analyzer Software </string>
  </property>
  <property name="windowOpacity">
   <double>42.000000000000000</double>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QSplitter" name="splitter_2">
        <property name="cursor">
         <cursorShape>ArrowCursor</cursorShape>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="opaqueResize">
         <bool>true</bool>
        </property>
        <property name="handleWidth">
         <number>10</number>
        </property>
        <property name="childrenCollapsible">
         <bool>false</bool>
        </property>
        <widget class="QTreeView" name="sampleTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>507</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>300</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
        <widget class="QWidget" name="verticalLayoutWidget">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="cursor">
             <cursorShape>ArrowCursor</cursorShape>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="opaqueResize">
             <bool>true</bool>
            </property>
            <property name="handleWidth">
             <number>10</number>
            </property>
            <property name="childrenCollapsible">
             <bool>false</bool>
            </property>
            <widget class="QTabWidget" name="param_tabs">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>100</verstretch>
              </sizepolicy>
             </property>
             <widget class="QWidget" name="tab">
              <attribute name="title">
               <string>Page</string>
              </attribute>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="textVisible">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar2">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>RightToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="font">
    <font>
     <strikeout>false</strikeout>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionToolbar_Import_SnP"/>
   <addaction name="actionRun"/>
   <addaction name="actionCalibrate_2"/>
   <addaction name="actionAlien"/>
   <addaction name="actionDeembed"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionImport_Project"/>
    <addaction name="actionImport_SnP"/>
    <addaction name="separator"/>
    <addaction name="actionSave_Project"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Excel"/>
    <addaction name="actionGenerate_Report"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
    <addaction name="separator"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionConfigure_Test"/>
    <addaction name="actionConfigure_Reports"/>
    <addaction name="separator"/>
    <addaction name="actionTest"/>
   </widget>
   <widget class="QMenu" name="menuVNA">
    <property name="title">
     <string>VNA</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionMeasure"/>
    <addaction name="actionCalibrate"/>
    <addaction name="separator"/>
    <addaction name="actionWho_am_I"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuVNA"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionImport_SnP">
   <property name="text">
    <string>Import SnP</string>
   </property>
  </action>
  <action name="actionImport_Project">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Import Project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionExport_Excel">
   <property name="text">
    <string>Export Excel</string>
   </property>
  </action>
  <action name="actionGenerate_Report">
   <property name="text">
    <string>Generate Report</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionConfigure_Test">
   <property name="text">
    <string>Configure Test</string>
   </property>
  </action>
  <action name="actionConnect_to_VNA">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionConfigure_Reports">
   <property name="text">
    <string>Configure Reports</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTest">
   <property name="text">
    <string>Test</string>
   </property>
  </action>
  <action name="actionToolbar_Import_SnP">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/import.png</normaloff>:/icons/import.png</iconset>
   </property>
   <property name="text">
    <string>Import Snp</string>
   </property>
   <property name="toolTip">
    <string>Import SnP file</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/new_project.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>Create New Project</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionWho_am_I">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Who am I ?</string>
   </property>
  </action>
  <action name="actionMeasure">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Measure</string>
   </property>
  </action>
  <action name="actionCalibrate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionRun">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/run.png</normaloff>
     <disabledoff>:/icons/run_disabled.png</disabledoff>
     <disabledon>:/icons/run_disabled.png</disabledon>:/icons/run.png</iconset>
   </property>
   <property name="text">
    <string>Run</string>
   </property>
  </action>
  <action name="actionCalibrate_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/calibrate.png</normaloff>
     <disabledoff>:/icons/calibrate_disabled.png</disabledoff>
     <disabledon>:/icons/calibrate_disabled.png</disabledon>:/icons/calibrate.png</iconset>
   </property>
   <property name="text">
    <string>Calibrate</string>
   </property>
  </action>
  <action name="actionAlien">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/alien.png</normaloff>:/icons/alien.png</iconset>
   </property>
   <property name="text">
    <string>Alien</string>
   </property>
   <property name="toolTip">
    <string>Alien</string>
   </property>
  </action>
  <action name="actionDeembed">
   <property name="icon">
    <iconset resource="../ressources/icons.qrc">
     <normaloff>:/icons/deembed.png</normaloff>:/icons/deembed.png</iconset>
   </property>
   <property name="text">
    <string>Deembed</string>
   </property>
   <property name="toolTip">
    <string>Deembed</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../ressources/icons.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>importSnp()</slot>
  <slot>newProject()</slot>
  <slot>setActiveSample()</slot>
  <slot>tabChange()</slot>
  <slot>connect()</slot>
  <slot>aquire()</slot>
  <slot>calibrate()</slot>
  <slot>disconnect()</slot>
  <slot>whoAmI()</slot>
  <slot>newAlien()</slot>
  <slot>newDeembed()</slot>
 </slots>
</ui>
